### PR TITLE
Fix menu button icon being stuck in a rotated position if no beatmap is playing

### PR DESCRIPTION
**Pull request by jorolf**

### CODE REVIEW

**Comment by peppy** [comment]
Pretty sure this won't fix the problem if the beatmap stops playing while a button is hovered

**Comment by jorolf** [comment]
I would've tested it, but there's no way to do that right now. _Probably should've done it anyway_

File: osu.Game/Graphics/Containers/BeatSyncedContainer.cs [hunk 1]
```diff
@@ -33,10 +33,17 @@ public class BeatSyncedContainer : Container
         /// </summary>
         public double TimeSinceLastBeat { get; private set; }
 
+        /// <summary>
+        /// If the beatmap is currently playing or if it has been stopped
+        /// </summary>
+        public bool BeatmapPlaying { get; private set; }
```
Comment (smoogipooo) [inline osu.Game/Graphics/Containers/BeatSyncedContainer.cs]: This should really be `AudioPlaying`.

**Review by smoogipoo — CHANGES_REQUESTED** [review]
As stated, this won't fix the case where music is stopped while the button is hovered, and also will break if the music starts while the button is hovered.

Closing this for now until further testing/implementation is done.

**Pull request closed without merging**